snmp/tests/test_e2e_core.py

```diff
@@ -97,6 +97,51 @@ def assert_apc_ups_metrics(dd_agent_check, config):
     aggregator.assert_all_metrics_covered()
 
 
+def test_e2e_memory_cpu_f5_big_ip(dd_agent_check):
```

Comment:
I think it will help (against future regression) to also have coverage for other metrics added in profiles:
- snmp/datadog_checks/snmp/data/profiles/_aruba-switch-cpu-memory.yaml
- snmp/datadog_checks/snmp/data/profiles/_checkpoint-firewall-cpu-memory.yaml
- snmp/datadog_checks/snmp/data/profiles/_cisco-cpu-memory.yaml
- snmp/datadog_checks/snmp/data/profiles/_fortinet-fortigate-cpu-memory.yaml
- snmp/datadog_checks/snmp/data/profiles/_generic-host-cpu-memory.yaml

Reply:
Planned on covering all those metrics once https://github.com/DataDog/datadog-agent/pull/11438 as it will add support for `scale_factor` thus changing the reported value.